README.md

```diff
@@ -44,16 +44,21 @@ For more details, see [building from source](https://github.com/Xpra-org/xpra/tr
 ---
 
 # Usage
+## Initial requirements
+`xpra` must be installed on the client and the host (version matters).  
+
+You can use the html5 client in which case xpra is only required on the host.  People that are using the html5 client would already be somewhat experienced xpra users. 
```

Comment:
Please remove "People..." - this is not helpful or true.

Reply:
I've removed it as you don't believe it is helpful.